cs/extras.xliff

```diff
@@ -0,0 +1,327 @@
+      <trans-unit id="languages.pa_IN">
+        <source>Punjabi</source>
+        <target>paňdžábština</target>
+    </trans-unit>
+      <trans-unit id="languages.pl">
+        <source>Polish</source>
+        <target>polština</target>
+    </trans-unit>
+      <trans-unit id="languages.pt_BR">
+        <source>Portuguese</source>
+        <target>portugalština</target>
+    </trans-unit>
+      <trans-unit id="languages.pt_PT">
+        <source>European Portuguese</source>
+        <target>portugalština (Evropa)</target>
+    </trans-unit>
+      <trans-unit id="languages.ru">
+        <source>Russian</source>
+        <target>ruština</target>
+    </trans-unit>
+      <trans-unit id="languages.sk">
+        <source>Slovak</source>
+        <target>slovenština</target>
+    </trans-unit>
+      <trans-unit id="languages.sl">
+        <source>Slovenian</source>
+        <target>slovinština</target>
+    </trans-unit>
+      <trans-unit id="languages.sq">
+        <source>Albanian</source>
+        <target>albánština</target>
+    </trans-unit>
+      <trans-unit id="languages.sv_SE">
+        <source>Swedish</source>
+        <target>švédština</target>
+    </trans-unit>
+      <trans-unit id="languages.tr">
+        <source>Turkish</source>
+        <target>turečtina</target>
+    </trans-unit>
+      <trans-unit id="languages.uk">
+        <source>Ukrainian</source>
+        <target>ukrajinština</target>
+    </trans-unit>
+      <trans-unit id="languages.zh_CN">
+        <source>Simplified Chinese</source>
+        <target>čínština (zjednodušená)</target>
+    </trans-unit>
+      <trans-unit id="languages.zh_TW">
+        <source>Traditional Chinese</source>
+        <target>čínština (tradiční)</target>
+    </trans-unit>
+    </body>
+  </file>
+  <file original="../src/apps/vpn/ui/screens/home/ViewServers.qml" datatype="plaintext" source-language="en" target-language="cs">
+    <body>
+      <trans-unit id="servers.al">
+        <source>Albania</source>
+        <target>Albánie</target>
+    </trans-unit>
+      <trans-unit id="servers.au">
+        <source>Australia</source>
+        <target>Austrálie</target>
+    </trans-unit>
+      <trans-unit id="servers.be">
+        <source>Belgium</source>
+        <target>Belgie</target>
+    </trans-unit>
+      <trans-unit id="servers.bg">
+        <source>Bulgaria</source>
+        <target>Bulharsko</target>
+    </trans-unit>
+      <trans-unit id="servers.br">
+        <source>Brazil</source>
+        <target>Brazílie</target>
+    </trans-unit>
+      <trans-unit id="servers.ca">
+        <source>Canada</source>
+        <target>Kanada</target>
+    </trans-unit>
+      <trans-unit id="servers.ch">
+        <source>Switzerland</source>
+        <target>Švýcarsko</target>
+    </trans-unit>
+      <trans-unit id="servers.co">
+        <source>Colombia</source>
+        <target>Kolumbie</target>
+    </trans-unit>
+      <trans-unit id="servers.cz">
+        <source>Czech Republic</source>
```

Comment:
Do we want to fix this?
https://en.wikipedia.org/wiki/Name_of_the_Czech_Republic

Reply:
There are a bunch of little inconsistencies like this in the translations we have right now. I'd say fixing them is outside of the cope of this PR.

Reply:
Sounds good to me. Keep in mind that you're going to need new IDs to fix them once we expose them, and a system to map old/new IDs in the code.